group_project_v2/stage/base.py

```diff
@@ -13,15 +13,15 @@
 from group_project_v2.api_error import ApiError
 from group_project_v2.mixins import (
     CommonMixinCollection, XBlockWithUrlNameDisplayMixin, AdminAccessControlXBlockMixin,
-    DashboardXBlockMixin, DashboardRootXBlockMixin
-)
+    DashboardXBlockMixin, DashboardRootXBlockMixin,
+    AuthXBlockMixin)
```

Comment:
@jbzdak nit: I believe it would fit on the previous line + closing paren on new line